device/bgpv6.yaml

```diff
@@ -3,53 +3,22 @@ components:
     Device.Bgpv6:
       x-status: under-review
       x-include: 
-      - '../common/common.yaml#/components/schemas/Named.Object'
+      - './bgp.yaml#/components/schemas/Device.Bgp'
       description: >-
-        Emulated BGPv4 router and routes
+        Container for BGPv6 peer settings and routes.
       type: object
       properties:
-        router_id:
+        local_address:
           description: >-
-            The BGP router identifier. 
-            It must be the string representation of an IPv4 address 
-          $ref: './patterns.yaml#/components/schemas/Device.Pattern'
-        as_number:
-          description: >-
-            Autonomous system (AS) number of 4 byte
-          $ref: './patterns.yaml#/components/schemas/Device.Pattern'
-        as_type:
-          description: >-
-            The type of BGP autonomous system.
-            External BGP (EBGP) is used for BGP links between two or more
-            autonomous systems.
-            Internal BGP (IBGP) is used within a single autonomous system.
+            Local IPv6 address of the emulated router
```

Comment:
It must match the IPv6 address of the parent under which this BGP v6 peer is configured.